vault/token_store.go

```diff
@@ -1325,29 +1339,69 @@ func (ts *TokenStore) handleUpdateLookupAccessor(req *logical.Request, data *fra
 // the token associated with the accessor
 func (ts *TokenStore) handleUpdateRevokeAccessor(req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
 	var urlaccessor bool
-	accessor := data.Get("accessor").(string)
-	if accessor == "" {
-		accessor = data.Get("urlaccessor").(string)
-		if accessor == "" {
+	accessors := data.Get("accessor").([]string)
+
+	if len(accessors) == 0 {
+		accessors = []string{data.Get("urlaccessor").(string)}
+		if len(accessors) == 0 {
 			return nil, &logical.StatusBadRequest{Err: "missing accessor"}
 		}
 		urlaccessor = true
 	}
 
-	aEntry, err := ts.lookupByAccessor(accessor, true)
-	if err != nil {
-		return nil, err
+	errs := make([]error, len(accessors))
+	tokens := make([]string, len(accessors))
```

Comment:
This list may not be the length of the accessor list in the case of errors.  I would just set it to length zero and append to the slice.